extensions/functions_logarithmic.yaml

```diff
@@ -0,0 +1,63 @@
+%YAML 1.2
+---
+scalar_functions:
+  -
+    name: "ln"
```

Comment:
It would be good to document what these options apply to (and their meaning) somewhere.  In arrow, arithmetic functions overflow by default.  However, logrithmic functions go to `-inf` or `NaN`.  I'm not sure if this latter behavior is "saturate" or "silent".